src/arcsf/data/tofu.py

```diff
@@ -0,0 +1,193 @@
+import math
+
+import numpy as np
+from datasets import Dataset, load_dataset
+from sklearn.model_selection import train_test_split
+
+
+def _flatten_list_of_lists(list: list[list]) -> list:
+    return sum(list, [])
+
+
+# returns the question index in the dataset, given an author number and question
+# number (within the author) (inner function)
+def _get_forget_index(
+    author: int,
+    question: int,
+    q_per_author: int,
+) -> int:
```

Comment:
Short enough to fit on one line:

```suggestion
def _get_forget_index(author: int, question: int, q_per_author: int) -> int:
```

also some other functions in this file (black will refactor them if you remove the comma after the last arg)

Reply:
Fixed